rakelib/license.rake

```diff
@@ -0,0 +1,45 @@
+# we need to call exit explicity  in order to set the proper exit code, otherwise
+# most common CI systems can not know whats up with this tests.
+require_relative "default_plugins"
+
+namespace "license" do
+
+  desc "run core specs"
+  #task "core" =>  ["test:setup"] do
+  task "core" do
+    require 'yaml'
+    generate_license_information
+  end
+
+end
+
+def all_installed_gems
+  Gem::Specification.all = nil
```

Comment:
I'm wondering why do you need to do that? Why not just using grasping all runtime dependencies the same way as https://github.com/elastic/logstash/blob/master/spec/license_spec.rb#L33 is doing? can you educate me of the differences?